### PR TITLE
feat(linux): add onboarding flow controller and login UI

### PR DESCRIPTION
**Pull request by R-Gld**
<details>
<summary>Description en français</summary>

## Résumé
Cette PR introduit la première partie du flux d'onboarding Linux v4 : un contrôleur de flux C++ (`OnboardingFlowController`), les vues QML associées (fenêtre d'onboarding, écran de connexion, animations Lottie), ainsi que le support de Qt 6.11.1 dans les recettes Conan.

## Changements
- Ajout de `OnboardingFlowController` : machine à états QObject exposant les étapes (`Login`, `DriveSelection`, `Synchronization`, `Ready`), l'état de connexion et les invokables QML (`requestLogin`, `requestAccountCreation`, `cancel`, `restart`).
- Ajout de `AppConstants::Login::signupUri()` dans `appconstants.h` pour centraliser l'URL d'inscription.
- Intégration du contrôleur dans `AppClientLinux` : connexion des signaux OAuth (`loginTokenSucceeded`, `loginTokenFailed`), gestion du `pendingOnboardingUserDbId` pour les cas où le cache utilisateur n'est pas encore disponible, et exposition au moteur QML via `setContextProperty`.
- Remplacement du placeholder de test dans `Main.qml` par `OnboardingWindow`, avec dimensions de fenêtre et couleur de fond appropriées.
- Ajout de `OnboardingWindow.qml` : disposition deux colonnes (contenu 63 % / illustration 37 %) avec fond `onboardingSurfacePrimary`.
- Ajout de `LoginView.qml` : en-tête de bienvenue, boutons "Créer un compte" et "Se connecter", texte de statut de connexion (état d'attente ou erreur).
- Ajout de `OnboardingAnimationsView.qml` : charge `LoaderStrokeDarkAnimation` ou `LoaderStrokeLightAnimation` selon le mode sombre, mis à l'échelle depuis les dimensions sources définies dans `IKOnboarding`.
- Ajout du singleton `IKOnboarding.qml` avec les constantes de mise en page et les dimensions des animations Lottie.
- Ajout des couleurs `onboardingSurfacePrimary` et `onboardingSurfaceSecondary` dans `IKColors.qml`, et de la teinte `neutralBlue900` pour le fond sombre.
- Ajout des fichiers `.lottie` (dark/light) pour les animations : check, loader-stroke, offline, pause, sync, syncro-file, disk-access-on, extension-on.
- Support de Qt 6.11.1 dans `conanfile.py` et `config.yml` (module Lottie Linux, sous-dossier d'installation, mappings Windows MSVC 2022).
- Réduction de la taille de `crashpad_handler` Linux via `strip --strip-all` dans la recette Sentry (les symboles DWARF sont intégrés dans le binaire ELF par défaut, inutiles au runtime).

## Détails techniques
`OnboardingFlowController` adopte le même contrat d'interface que la version macOS : il gère uniquement l'état de navigation (étape courante, état de connexion) et délègue les actions métier (OAuth, chargement des drives) aux services existants. Le cas de synchronisation du cache utilisateur - où le token OAuth arrive avant que `AppCache::users` ne soit mis à jour - est géré par un `_pendingOnboardingUserDbId` optionnel, résolu dès que le signal `usersChanged` est émis.

## Notes
Le lancement OAuth (`loginRequested`) et l'ouverture de l'URL de création de compte ne sont pas encore implémentés sur Linux ; des warnings de log le signalent explicitement. Ils feront l'objet de PRs suivantes.

</details>

---

## Summary
This PR introduces the first part of the Linux v4 onboarding flow: a C++ `OnboardingFlowController` state machine, the associated QML views (onboarding window, login screen, Lottie animations), and Qt 6.11.1 support in the Conan recipes.

## Changes
- Add `OnboardingFlowController`: a QObject state machine exposing steps (`Login`, `DriveSelection`, `Synchronization`, `Ready`), login state, and QML-invokable actions (`requestLogin`, `requestAccountCreation`, `cancel`, `restart`).
- Add `AppConstants::Login::signupUri()` in `appconstants.h` to centralize the sign-up URL.
- Wire `OnboardingFlowController` into `AppClientLinux`: connect OAuth signals (`loginTokenSucceeded`, `loginTokenFailed`), handle the `pendingOnboardingUserDbId` edge case where the user cache is not yet populated when the token arrives, and expose the controller to the QML engine via `setContextProperty`.
- Replace the placeholder test rectangle in `Main.qml` with `OnboardingWindow`, and set appropriate window dimensions and background color.
- Add `OnboardingWindow.qml`: two-column layout (content 63% / illustration 37%) with `onboardingSurfacePrimary` background.
- Add `LoginView.qml`: welcome heading, "Create account" and "Log in" buttons, and a login status text area that reflects waiting and error states.
- Add `OnboardingAnimationsView.qml`: loads `LoaderStrokeDarkAnimation` or `LoaderStrokeLightAnimation` depending on theme mode, scaled from source dimensions defined in `IKOnboarding`.
- Add `IKOnboarding.qml` singleton with layout ratios and Lottie animation dimension constants.
- Add `onboardingSurfacePrimary` and `onboardingSurfaceSecondary` color tokens to `IKColors.qml`, and the `neutralBlue900` dark-mode shade.
- Add `.lottie` asset files (dark/light variants) for: check, loader-stroke, offline, pause, sync, syncro-file, disk-access-on, extension-on.
- Support Qt 6.11.1 in `conanfile.py` and `config.yml` (Linux Lottie module, install subfolder, Windows MSVC 2022 mappings).
- Strip debug symbols from `crashpad_handler` on Linux in the Sentry recipe via `strip --strip-all` to reduce binary size (DWARF info is embedded in ELF by default but is not needed at runtime).

## Technical Details
`OnboardingFlowController` mirrors the macOS onboarding interface contract: it owns only navigation state (current step, login state) and delegates business actions (OAuth, drive loading) to existing services. The user-cache race condition - where the OAuth token arrives before `AppCache::users` is updated - is handled via an optional `_pendingOnboardingUserDbId` member, resolved as soon as the `usersChanged` signal fires.

## Notes
The OAuth launcher (`loginRequested`) and account creation URL opening are not yet implemented on Linux; explicit log warnings mark these stubs. They will be addressed in follow-up PRs.

### CODE REVIEW

File: src/gui4/linux/ui/tokens/IKColors.qml [hunk 1]
```diff
@@ -92,6 +93,8 @@ QtObject {
     readonly property color surfacePrimary: darkMode ? _p.neutralBlue800 : _p.neutralBlue50
     readonly property color surfaceSecondary: darkMode ? _p.neutralBlue700 : _p.neutralBlue100
     readonly property color surfaceTertiary: darkMode ? _p.neutralBlue600 : _p.neutralBlue200
+    readonly property color onboardingSurfacePrimary: darkMode ? _p.neutralBlue900 : "#FFFFFF"
+    readonly property color onboardingSurfaceSecondary: darkMode ? _p.neutralBlue700 : "#F7F7F7"
```
Comment (herve-er) [inline src/gui4/linux/ui/tokens/IKColors.qml]: Seems like #FFFFFF and #F7F7F7 could be in  a Primitives

**Review by herve-er — COMMENTED** [review]
LGTM, just a remark about translations. It could be a good idea to tackle this issue before going too far with the implementation, as it might be painful to change all the `tr()` calls later if they need to be compatible with Loco

File: src/gui4/linux/ui/tokens/IKColors.qml (hunk 1 above)
Comment (R-Gld) [reply src/gui4/linux/ui/tokens/IKColors.qml]: Yep, totally, done here: https://github.com/Infomaniak/desktop-kDrive/pull/1960/commits/4ff2b3d942d25d3588ae4243de54e951bcf4f45e
